### PR TITLE
dvc 3.0.0

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by skshetry** [comment]
@chenrui333, do you mind updating the path from `dvc/utils/pkg.py` to `dvc/_build.py`? We changed the path in 3.0.

https://github.com/Homebrew/homebrew-core/blob/edc9313d1922b97a1dc7c3a21f8585efc4c40ba0/Formula/dvc.rb#L770

```rb
    File.write("dvc/_build.py", "PKG = \"brew\"")
```

**Comment by chenrui333** [comment]
> @chenrui333, do you mind updating the path from `dvc/utils/build.py` to `dvc/_build.py`? We changed the path in 3.0.
> 
> https://github.com/Homebrew/homebrew-core/blob/edc9313d1922b97a1dc7c3a21f8585efc4c40ba0/Formula/dvc.rb#L770
> 
> ```ruby
>     File.write("dvc/_build.py", "PKG = \"brew\"")
> ```

Can you link the related commits or PRs in here as well? I am updating the PR now.

File: Formula/dvc.rb [hunk 1]
```diff
@@ -4,10 +4,9 @@ class Dvc < Formula
   desc "Git for data science projects"
   homepage "https://dvc.org"
   # TODO: Check if we can use unversioned `protobuf` at version bump
```
Comment (p-linnane) [inline Formula/dvc.rb:6]: We need to check this next time. Don't want to waste the long build run we've already done this time around.

**Review by p-linnane — APPROVED** [review]

File: Formula/dvc.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/dvc.rb:6]: 👍

**Pull request merged**